test_supabase_connection.py

```
@@ -0,0 +1,151 @@
+        required_buckets = ["route-images", "model-outputs"]
+        found_buckets = []
+        missing_buckets = []
+
+        for bucket_name in required_buckets:
+            try:
+                # Try to access the bucket
+                bucket_proxy = client.storage.from_(bucket_name)
+                bucket_proxy.list()  # This will raise if bucket doesn't exist
+                found_buckets.append(bucket_name)
+                print(f"       [OK] {bucket_name} - accessible")
+            except Exception as e:
```

Pylint (4.0.4)

[warning] Catching too general exception Exception

(W0718)